openshift_scalability/pvc_delete.sh

```diff
@@ -0,0 +1,81 @@
+#/!/bin/bash
```

Comment:
@paigerube14 wondering why you decided to use bash shell inside a python directory instead of creating a binding python script instead.

Reply:
For the most part I've seen that the scripts in this repo are in bash. I also was incorporating a smaller pre-existing bash script into this one to create a fully encompassed test case.
I could create it in python. Thoughts?

Reply:
@paigerube14 others can chime in.  IMO it's just harder to debug with bash when something goes wrong (could just be my bash-fu is horrible).